mono/metadata/monitor.c

```diff
@@ -791,6 +791,10 @@ signal_monitor (gpointer mon_untyped)
 	mono_coop_mutex_unlock (mon->entry_mutex);
 }
 
+#if ENABLE_NETCORE
```

Comment:
Do we need to use this if we're building without DISABLE_PERFCOUNTERS?
Also, nit: current pattern is `#ifdef ENABLE_NETCORE`.

Reply:
Would be nice to switch to `#if` and then where possible, often, to `if`.